doc/cephfs/fs-volumes.rst

```diff
@@ -130,16 +130,24 @@ its parent directory and no size limit.
 
 Remove a subvolume using::
 
-    $ ceph fs subvolume rm <vol_name> <subvol_name> [--group_name <subvol_group_name> --force]
+    $ ceph fs subvolume rm <vol_name> <subvol_name> [--group_name <subvol_group_name> --force --retain-snapshots]
```

Comment:
nit: the more canonical format would be this:

```suggestion
    $ ceph fs subvolume rm <vol_name> <subvol_name> [--group_name <subvol_group_name>] [--force] [--retain-snapshots]
```

Reply:
Addressed (also in a few other places in the same rst file)